src/Persistent/Pure.hs

```diff
@@ -425,6 +427,23 @@ findUser = findInById dbUserMap
 getUsers :: Query [User]
 getUsers = view dbUsers
 
+getUsersInClass :: SchoolClass -> Query [User]
+getUsersInClass clss = filter (isClassInRole clss . view userRole) <$> view dbUsers
```

Comment:
There is a similar query:
https://github.com/liqd/aula/blob/master/src/Persistent/Idiom.hs#L34